### PR TITLE
Log frontend perf results in codehealth

### PR DESCRIPTION
**Pull request by youknowriad**
## What?

In #47037 some frontend performance metrics have been added to our performance job. This PR logs the results in code health so we can track them over time.

I've did some manual tests locally, but we won't be able to test this 100% until we merge it :)

### CODE REVIEW

File: bin/log-perormance-results.js [hunk 1]
```diff
@@ -6,12 +6,27 @@
...
+const { mapKeys } = require( 'lodash' );
 const [ token, branch, hash, baseHash, timestamp ] = process.argv.slice( 2 );
 
-const performanceResults = JSON.parse(
-	fs.readFileSync(
-		path.join( __dirname, '../post-editor-performance-results.json' ),
-		'utf8'
+const resultsFiles = [
+	{
+		file: 'post-editor-performance-results.json',
```
Comment (oandregal) [inline bin/log-perormance-results.js:14]: I was trying to run this locally. I commented the parts that were about making the request to codehealth. Then, what I did was running the performance test and move the generated files to where this command expects them to be. For example, for post editor:

- `npm run test:performance -- --puppeteer-interactive packages/e2e-tests/specs/performance/post-editor.test.js`
- `cp packages/e2e-tests/specs/performance/post-editor.test.results.json post-editor-performance-results.json`

And similarly for the other two. Note that I had to rename the file, so I wonder if I missed another command? I guess my question is: are these paths expected?

Other than that, the command reported numbers, so it seems fine.

**Review by oandregal — APPROVED** [review]
Left a comment about the paths https://github.com/WordPress/gutenberg/pull/47442/files#r1087679545 but other than that it seems fine.

File: bin/log-perormance-results.js (hunk 1 above)
Comment (youknowriad) [reply bin/log-perormance-results.js:14]: yeah, you need to run the comparison command `./bin/plugin/cli.js perf branch1 branch2` to get the right files.

**Review by tyxla — COMMENTED** [review]
@youknowriad considering that we already removed all instances of `mapKeys()` in the repository, WDYT about replacing `_.mapKeys()` with `Object.entries().map()`?

**Comment by youknowriad** [comment]
@tyxla sure, this is a tool though so there's no bundle size issue or anything. It would be good to start adding some lint errors whenever we get rid of some import.

**Comment by tyxla** [comment]
> @tyxla sure, this is a tool though so there's no bundle size issue or anything. 

Sure, I realize that but ideally our end goal is to get rid of it entirely from the repo, otherwise, it can quickly be reintroduced if it sticks as a dependency. 

> It would be good to start adding some lint errors whenever we get rid of some import.

We're already [doing that](https://github.com/WordPress/gutenberg/blob/trunk/.eslintrc.js#L45-L157), but for CommonJS (`require()`) it doesn't work, since we'd have to forbid the entire `lodash` module, and there are still quite a few usages.

**Comment by tyxla** [comment]
Alternatively, for CJS we could start importing from specific modules (e.g. `require( 'lodash/some' )` and forbid the global `lodash` import, if you think that will be better.

**Pull request merged**